app/physician-bio/[slug]/page.tsx

```diff
@@ -1,11 +1,149 @@
 }
 
-export default function PhysicianBio({ params }: { params: { slug: string } }) {
+const ContentParagraph = ({ contentItem }: { contentItem: any }) => {
+  return <div>{renderRichTextToReactComponent(contentItem as Document)}</div>;
+};
+
+const Section = ({ section }: { section: BioPageSectionType }) => (
+  <div key={section.sys.id}>
+    <h2 className="text-xl">{section.fields.title}</h2>
+    <div className="text-sm">
+      {section.fields.content.content.map(
+        (
+          contentItem: { content: Array<{ value: string }> },
+          contentIndex: number,
+        ) => (
+          <ContentParagraph key={contentIndex} contentItem={contentItem} />
+        ),
+      )}
+    </div>
+  </div>
+);
+
+export default async function PhysicianBio({
+  params,
+}: {
+  params: { slug: string };
+}) {
+  const docBio = await getPhysicianBioBySlug(params.slug);
+
+  let educationAndCertificatesHeaderRendered = false;
+  let researchInsightsAndPublicationsHeaderRendered = false;
   return (
-    <main>
-      <h1>Physician Bio - {params.slug} </h1>
+    <main className="m-10">
+      <div>
+        <Image
+          src={docBio.physicianPortrait.fields.file.url}
+          alt=""
+          width={docBio.physicianPortrait.fields.file.details.image.width}
+          height={docBio.physicianPortrait.fields.file.details.image.height}
+        />
+        <h1 className="text-lg text-[#800080]">{docBio.physicianName}</h1>
+        <h3 className="text-base">Specializes in:</h3>
+        <div>
+          {renderRichTextToReactComponent(
+            docBio.specialties as unknown as Document,
+          )}
+        </div>
+        <div id="phone-numbers">
+          <p>Appointment Number: {docBio.appointmentNumber}</p>
+          <p>Physician Number: {docBio.physicianNumber}</p>
+        </div>
+      </div>
+      <div className="flex items-center">
+        <hr className="flex-grow border-[#99C221]"></hr>
+      </div>
+      <div id="overview">
+        <h2 className="text-xl text-[#0076AD]">Overview</h2>
+      </div>
+      <div id="asset-container">
+        {Array.isArray(docBio.asset) && docBio.asset.length > 0 && (
+          <div>
+            {docBio.asset.map((asset: AssetType) => (
+              <div key={asset.sys.id}>
+                <h3>{asset.fields.title}</h3>
+                <Image
+                  src={asset.fields.file.url}
+                  alt={asset.fields.description}
+                  width={asset.fields.file.details.image.width}
+                  height={asset.fields.file.details.image.height}
+                />
+              </div>
+            ))}
+          </div>
+        )}
+      </div>
+
+      <div id="temp-bio-page-sections">
+        {docBio.bioPageSection.map((section: BioPageSectionType) => (
+          <div key={section.sys.id}>
+            {!educationAndCertificatesHeaderRendered &&
+              (section.fields.title === "Medical School" ||
```

Comment:
I think we can clean this conditional logic up a bit. General practice to avoid too many conditionals in the the markup.